### ISSUE TITLE
EBS volumes don't store KmsKeyId 

### ISSUE THREAD

**Issue by tekumara** [spulec/moto#3248]
```python
from moto import mock_ec2
import boto3

mock = mock_ec2()
mock.start()

resource = boto3.resource("ec2")

kwargs = {
    "AvailabilityZone": "ap-southeast-2a",
    "Size": 8,
    "Encrypted": True,
    "KmsKeyId": "key"    
}

volume = resource.create_volume(**kwargs)

print(volume.kms_key_id)
```

The above prints `None`. I was expecting it to print `key`.

moto 1.3.14

**Comment by bblommers** [comment]
Thanks for raising this @tekumara - marking this as an enhancement

**Comment by bblommers** [comment]
This is now available as of moto >= 1.3.16.dev65
